docs/docs/FAQ/Privacy-FAQ.md

```diff
@@ -0,0 +1,14 @@
+---
+sidebar_position: 9
+---
+import TOCInline from '@theme/TOCInline';
+
+# Privacy
+
+### Is all data local?
+Everything stays local, the models get downloaded from a repository the first time you run it.
+Machine learning is also done locally,
+Machine learning URL isn't pointing to the external page. It is the DNS of the container within the docker's network context.
+Model retrievals are done automatically from the [hugging face repository](https://huggingface.co/immich-app). more info [here](https://www.reddit.com/r/immich/comments/17obeyj/is_all_data_local/). 
```

Comment:
Not that this is wrong, but we need to be more precise with the wording. I think the answer to this question should be in the form of a privacy policy and a TL;DR linking to it. Probably better to add this separately.

Reply:
I think it's worth mentioning somewhere, I've seen such a question repeated several times on Reddit.

Because currently Immich does not have a section that talks about a privacy policy and according to Alex's words on Reddit, I don't know if when there is a privacy policy what information will be included in it if it is so short?
I mean it should be something like.. we don't collect anything

Reply:
I know what you mean, but I just want to be sure we're not exposing ourselves legally here. I have some reservations about a blanket statement like this.

Are we 100% sure there's nothing being sent, even indirectly? For instance, what if we have a dependency that defaults to sending telemetry? Even if we don't right now, what if this happens in the future?

What does local mean? Different services can run in different locations, and we share data across these services. More broadly, given how many environments Immich can be deployed in, what do we include and exclude when we're talking about data being shared? Does inserting into a cloud provider's database count as sharing with a third party?

Do requests to e.g. GitHub or Hugging Face need to be mentioned? We're technically sharing IP address with them.

We also have OAuth integration, so we'd need to mention that too.

Reply:
Yes I understand that completely, in fact it is a big issue that requires understanding on the subject, I guess Immich could draw some essential knowledge from the [Photoprism](https://www.photoprism.app/privacy) project regarding the matter of privacy policy and implement an adapted privacy policy accordingly.

I will remove this section.